### ISSUE TITLE
Error when exporting server logs

### ISSUE THREAD

**Issue by garethbowen** [medic/medic-webapp#1027]
Marc and Sharon report exporting Server Logs fails with this in the api log:

```
2015-06-23T03:33:54.697Z - info: 127.0.0.1 - sharon [23/Jun/2015:03:33:54 +0000] "GET /api/v1/export/logs?format=zip&locale=en HTTP/1.0" - - "http://medic.local/medic/_design/medic/_rewrite" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.118 Safari/537.36"
 ```

In the browser console: `502 Bad Gateway`

It takes a couple of minutes to die so I suspect it's a timeout in API somewhere. Either speed up the export or increase the timeout.

**Comment by browndav** [comment]
Interesting; almost like it's just blocking instead of doing useful log-collection work. I'll take a look today.

**Comment by browndav** [comment]
I'm getting a total run time of about 3 seconds after a default install and first-run of the latest image. If you're able, could you switch to virtual terminal six in the VMware window (Ctrl-Alt-Fn-F6 IIRC), run `sudo /boot/print-logs | wc -c` and let me know what it prints?

**Comment by browndav** [comment]
Okay, that's about a two order of magnitude improvement. Suspect that'll fix what you're seeing, but the `wc` output would still be helpful. New images on the way; I'll post the link here when they're up.

**Comment by browndav** [comment]
Try http://dev.medicmobile.org/downloads/medic-os/20150623/.

**Comment by sglangevin** [comment]
@browndav - sorry - missed your earlier request. I tried doing that and kinda of messed up my VM and then that command wasn't recognized, I probably missed a step. Anyway, I've downloaded the new iso to test and tried updating my DIY version on the Windows computer in the office using dashboard settings. The logs downloaded just fine on my Mac but don't seem to be working on my updated DIY (not sure if they even should work there).
